server/lib/weather/weather.command.js

```diff
@@ -1,38 +1,90 @@
+const dayjs = require('dayjs');
 const logger = require('../../utils/logger');
 const { ServiceNotConfiguredError, NoValuesFoundError } = require('../../utils/coreErrors');
+const { INTENTS } = require('../../utils/constants');
+const { NoWeatherFoundError } = require('./weather.error');
+const { NotFoundError } = require('../../utils/coreErrors');
+
+/**
+ * @description Capitalize First Letter.
+ * @param {string} string - The word.
+ * @returns {string} The capitalized first letter word.
+ * @example
+ * capitalizeFirstLetter('word');
+ */
+function capitalizeFirstLetter(string) {
+  return string.charAt(0).toUpperCase() + string.slice(1);
+}
 
 /**
  * @description Get the weather in a text request.
  * @param {Object} message - The message sent by the user.
  * @param {Object} classification - The classification calculated by the brain.
  * @param {Object} context - The context object containing found variables in question.
+ * @returns {Object} Nothing.
  * @example
  * weather.command(message, classification, context);
  */
 async function command(message, classification, context) {
-  let weather;
   try {
     const houses = await this.house.get();
     const house = houses[0];
     if (!house || !house.latitude || !house.longitude) {
       throw new NoValuesFoundError();
     }
-    switch (classification.intent) {
-      case 'weather.get':
-        weather = await this.get(house);
+    const weather = await this.get(house);
+
+    if (`intent.${classification.intent}` === INTENTS.WEATHER.GET) {
+      const dateEntity = classification.entities.find((entity) => entity.entity === 'date');
+
+      if (dateEntity === undefined) {
+        // current day
         context.temperature = weather.temperature;
         context.units = weather.units === 'metric' ? '°C' : '°F';
-        await this.messageManager.replyByIntent(message, `weather.get.success.${weather.weather}`, context);
-        break;
-      default:
-        throw new Error('Not found');
+
+        await this.messageManager.replyByIntent(message, `weather.get.success.now.${weather.weather}`, context);
+      } else {
+        let diff;
+        if (dateEntity.resolution.type === 'interval') {
+          diff = dayjs(dateEntity.resolution.strFutureValue).diff(dayjs().startOf('day'), 'day');
+        } else {
+          diff = dayjs(dateEntity.resolution.date).diff(dayjs().startOf('day'), 'day');
```

Comment:
Looks very close to line 49 `dayjs(myvar).diff(dayjs().startOf('day'), 'day');`
Maybe the if can determine only if `strFutureValue` or `date` is used.